cpp/dolfinx/geometry/gjk.cpp

```diff
@@ -161,89 +208,88 @@ nearest_simplex(const xt::xtensor<double, 2>& s)
         }
       }
     }
-    return {std::move(smin), vmin};
+    return {smin, vmin};
   }
   default:
-  {
     throw std::runtime_error("Number of rows defining simplex not supported.");
   }
-  }
 }
 //----------------------------------------------------------------------------
-// Support function, finds point p in bd which maximises p.v
-xt::xtensor_fixed<double, xt::xshape<3>>
-support(const xt::xtensor<double, 2>& bd,
-        const xt::xtensor_fixed<double, xt::xshape<3>>& v)
+// Helper function, finds point p in bd which maximises p.v
```

Comment:
It should be called the "Support function", as this is the name of it in the actual algorithm